pallets/fee/src/default_weights.rs

```diff
@@ -1,33 +1,37 @@
 
-//! Autogenerated weights for fee
+//! Autogenerated weights for `fee`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-01-26, STEPS: `100`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `MacBook-Pro`, CPU: `<UNKNOWN>`
-//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+//! DATE: 2023-05-24, STEPS: `100`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Bs-MacBook-Pro.local`, CPU: `<UNKNOWN>`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
 
 // Executed Command:
 // ./target/release/spacewalk-standalone
 // benchmark
 // pallet
-// --chain=dev
-// --pallet=fee
-// --extrinsic=*
+// --chain
+// dev
+// --execution=wasm
```

Comment:
Wow, since when does it work with `wasm`? That's great. I remember when I created all the benchmarks a while ago this parameter was not working and unfortunately I was not able to figure out why. Great that it works now